scripts/validate.sh

```
@@ -117,15 +117,15 @@ if command -v fzf >/dev/null 2>&1; then
-  print_info "fzf not installed, skipping FZF_BASE check"
+  print_skip "fzf not installed, skipping FZF_BASE check"
 fi
 
-# Check if our PATH modifications are loaded
-DOTFILES_BIN_CHECK=$(zsh -c 'source ~/.zshrc 2>/dev/null; echo $PATH' | grep -c "$DOTFILES_DIR/bin")
-if [ "$DOTFILES_BIN_CHECK" -gt 0 ]; then
-  print_success "DOTFILES_DIR/bin is in PATH"
+# Check if ~/bin is in PATH (where stow symlinks bin scripts)
+HOME_BIN_CHECK=$(zsh -c 'source ~/.zshrc 2>/dev/null; echo $PATH' | grep -c "$HOME/bin" || true)
+if [ "$HOME_BIN_CHECK" -gt 0 ]; then
+  print_success "~/bin is in PATH"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)